Formula/spidermonkey.rb

```diff
@@ -1,19 +1,13 @@
 class Spidermonkey < Formula
   desc "JavaScript-C Engine"
   homepage "https://developer.mozilla.org/en/SpiderMonkey"
-  url "https://archive.mozilla.org/pub/mozilla.org/js/js185-1.0.0.tar.gz"
-  version "1.8.5"
-  sha256 "5d12f7e1f5b4a99436685d97b9b7b75f094d33580227aa998c406bbae6f2a687"
-  revision 1
+  url "https://people.mozilla.org/~sstangl/mozjs-38.2.1.rc0.tar.bz2"
+  version "38.2.1"
```

Comment:
I've submitted an update to this formula several times for (I think) a couple of years.  Usually it takes a very long time before someone reviews it and in the past it coincided with me being very busy at work and by the time I got around to replying to the comments someone had already closed the pull request due to inactivity so I had to resubmit and the cycle restarted.